src/components/Table/hoc/withTableActions/withTableActions.tsx

```diff
@@ -74,9 +77,103 @@ interface WithTableActionsState<I> {
     popupData: PopupData<I> | null;
 }
 
+const isActionGroup = <I extends TableDataItem>(
+    config: TableActionConfig<I>,
+): config is TableActionGroup<I> => {
+    return Array.isArray((config as TableActionGroup<I>).items);
+};
+
 const b = block('table');
+const actionsCn = b('actions');
+const BUTTON_CLASS_NAME = b('actions-button');
```

Comment:
Why this variable named in different case?

Reply:
59eae6957ab0033cf26fe3197bbe23bffb31ef2f